src/routes/home/Home.js

```diff
@@ -51,20 +51,21 @@ export default function HomePage() {
            * items in this section will feature clickable icons that will scroll to a
            * section in the page
            */}
-          <CategoryNav navItems={originalSections} />
-
+          {products.length > 0 ? (
+            <CategoryNav navItems={products[0]} />
```

Comment:
You cannot use `product[0]` as the category here. You can't know for sure which document is the category document based on their index position. The only way to know for sure is to check the document id: if the id is `categories`, then it is the category (check the document for correct information)